Dockerfile

```diff
@@ -8,6 +8,7 @@ COPY --chown=xmtp:xmtp rust-toolchain .
 ENV RUSTUP_PERMIT_COPY_RENAME "yes"
 
 RUN rustup update
+RUN curl -LsSf https://get.nexte.st/latest/linux | tar zxf - -C ${CARGO_HOME:-~/.cargo}/bin
```

Comment:
@insipx are we using the dev container?

Reply:
I don't use it myself, but unsure if anyone else does

Reply:
The readme file stated to use the vscode dev container, even when I had to run the nextest command locally I had to install it, so for easier developer access I included it in the docker file